package.json

```diff
@@ -22,12 +22,12 @@
   },
   "devDependencies": {
     "@arethetypeswrong/cli": "^0.15.1",
+    "@gadget-client/app-with-file-fields": "^1.5.0",
     "@gadget-client/app-with-no-user-model": "^1.10.0",
     "@gadget-client/bulk-actions-test": "^1.113.0",
     "@gadget-client/full-auth": "^1.9.0",
-    "@gadget-client/kitchen-sink": "1.4.0-development.184",
+    "@gadget-client/kitchen-sink": "1.5.0-development.200",
```

Comment:
I updated this real Gadget app to fix some of the legacy model namespace issues, so I updated the package version.